controllers/vrg_recipe.go

```diff
@@ -66,20 +74,20 @@ func RecipeElementsGet(ctx context.Context, reader client.Reader, vrg ramen.Volu
 
 func recipeVolumesAndOptionallyWorkflowsGet(ctx context.Context, reader client.Reader, vrg ramen.VolumeReplicationGroup,
 	ramenConfig ramen.RamenConfig, log logr.Logger, recipeElements *RecipeElements,
-	workflowsGet func(recipe.Recipe, *RecipeElements, ramen.VolumeReplicationGroup) error,
+	workflowsGet func(recipe.Recipe, *RecipeElements, ramen.VolumeReplicationGroup, ramen.RamenConfig) error,
 ) error {
 	if vrg.Spec.KubeObjectProtection == nil {
 		*recipeElements = RecipeElements{
-			PvcSelector: pvcSelectorDefault(vrg),
+			PvcSelector: getPVCSelector(vrg, ramenConfig, nil, nil),
 		}
 
 		return nil
 	}
 
 	if vrg.Spec.KubeObjectProtection.RecipeRef == nil {
 		*recipeElements = RecipeElements{
-			PvcSelector:     pvcSelectorDefault(vrg),
-			CaptureWorkflow: captureWorkflowDefault(vrg),
+			PvcSelector:     getPVCSelector(vrg, ramenConfig, nil, nil),
+			CaptureWorkflow: captureWorkflowDefault(vrg, ramenConfig),
 			RecoverWorkflow: recoverWorkflowDefault(),
 		}
 
```

Comment:
Comment on old code below, so a Recipe can be fetched from any namespace, that is neither the VRG NS or in the list of `ProtectedNamespaces`. Now, not sure that is intentional, hence starting a comment on the same.